build-android.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# exit this script if any commmand fails
+set -e
+#  
+
+host_os=`uname -s | tr "[:upper:]" "[:lower:]"`
```

Comment:
this build script was modified for clang, based on [dumganhar/build-android.sh](https://github.com/dumganhar/v8/blob/master/build-android.sh)